health_check/plugins.py

```diff
@@ -21,5 +21,12 @@ def register(self, plugin, **options):
         # Instantiate the admin class to save in the registry
         self._registry.append((plugin, options))
 
+    def reregister(self, replace_class, new_plugin, **options):
```

Comment:
How's this related?

Reply:
https://github.com/KristianOellegaard/django-health-check/pull/191/files#diff-88b99bb28683bd5b7e3a204826ead112R177

Reply:
Alternatively, we could add a sort of `def update_registry(self, class, option, new_value):`. Either way, we need a way to override the default value's of plugin attributes (critical flag, description, identifier, etc.)

Reply:
I wrote a comment further up how to not register plugins to begin with. There is no need for re `update` function.